Tests/SSL/test_ssl_containers.py

```diff
@@ -124,12 +128,12 @@ def test_innereye_ssl_container_cifar10_resnet_simclr() -> None:
     # Check the metrics that were recorded during training
     # Note: It is possible that after the PyTorch 1.10 upgrade, we can't get parity between local runs and runs on
     # the hosted build agents. If that suspicion is confirmed, we need to add branching for local and cloud results.
-    expected_metrics = {'simclr/val/loss': 2.8736939430236816,
-                        'ssl_online_evaluator/val/loss': 2.268489360809326,
+    expected_metrics = {'simclr/val/loss': 2.8797268867492676,
+                        'ssl_online_evaluator/val/loss': 2.272602081298828,
                         'ssl_online_evaluator/val/AccuracyAtThreshold05': 0.20000000298023224,
-                        'simclr/train/loss': 3.6261844635009766,
+                        'simclr/train/loss': 3.6261773109436035,
                         'simclr/learning_rate': 0.0,
-                        'ssl_online_evaluator/train/loss': 3.1140503883361816,
+                        'ssl_online_evaluator/train/loss': 3.1140334606170654,
                         'ssl_online_evaluator/train/online_AccuracyAtThreshold05': 0.0}
```

Comment:
for my own knowledge: was this due to a previous PyTorch update? nothing in this PR seems should affect these figures. If it's something that changes often maybe update the tolerance in _compare_stored_metrics?

Reply:
The reason for this is a slight change to randomization. We are now creating dataloaders one more time for diagnostics.